pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -189,8 +189,8 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 		WorkingDir:      config.VarLogCephDir,
 	}
 
-	container = config.ConfigureStartupProbe(cephv1.KeyMgr, container, c.spec.HealthCheck)
-	container = config.ConfigureLivenessProbe(cephv1.KeyMgr, container, c.spec.HealthCheck)
+	container = config.ConfigureStartupProbe(container, c.spec.HealthCheck.StartupProbe[cephv1.KeyMgr])
+	container = config.ConfigureLivenessProbe(container, c.spec.HealthCheck.LivenessProbe[cephv1.KeyMgr])
```

Comment:
How are we verifying in unit tests that this is getting the configured probe for the mgr and not getting a configured probe for mon or osd? I think we should test these probes in the `spec_test.go` to ensure.